crates/codegen/parser_runtime/src/templates/mod.tera

```diff
@@ -0,0 +1,14 @@
+#[macro_use]
```

Comment:
I wonder what is the difference between `crates/codegen/parser_runtime/src/templates/mod.tera` and `crates/codegen/parser_runtime/src/lib.rs`?